### ISSUE TITLE
batch_normalization layer argument name and documentation wrong in core and contrib layers

### ISSUE THREAD

**Issue by jonasrauber** [tensorflow/tensorflow#6736]
The batch normalization layers both in core and contrib take a `center` argument that controls whether a variable beta should be created that is then **added after the normalization**. The name of the argument (`center`) and even more so the documentation (_center: If True, subtract `beta`. If False, `beta` is ignored._) indicate however that this argument controls whether the mean of the input is subtracted as part of the normalization.

**Comment by yaroslavvb** [comment]
@sguada
